### DISCUSSION TITLE
How can the platform keyword in vcpkg.json be made to recognize operating systems other than common ones like Windows, Linux, and Android?

### DISCUSSION THREAD

**Discussion by AlexOct** [microsoft/vcpkg#39698] [Q&A] [answered]
I added a new operating system called "myos" in the triplet, and the basic project build is working. However, when introducing third-party libraries and setting the platform keyword to "myos" in vcpkg.json, vcpkg does not recognize the required libraries and prompts that the keyword is not available. Is there a way to make the platform keyword in vcpkg.json recognize "myos"?

**Comment by autoantwort** [comment]
I think you can set `VCPKG_DEP_INFO_OVERRIDE_VARS` in your triplet. So
`set(VCPKG_DEP_INFO_OVERRIDE_VARS "myos")`

**Reply by autoantwort** [reply, marked as answer]
But afaik you have to set `set(VCPKG_DEP_INFO_OVERRIDE_VARS "!myos")` in the others triplet or you get an error that the identifier `myos` is not known.

**Reply by AlexOct** [reply]
Is there any way to set it once in one place and use it elsewhere, or else it will require too many modifications?

**Reply by autoantwort** [reply]
Afaik no
